test/libsolidity/SMTCheckerTest.cpp

```diff
@@ -53,6 +53,14 @@ SMTCheckerTest::SMTCheckerTest(string const& _filename): SyntaxTest(_filename, E
 
 	if (m_enabledSolvers.none() || m_modelCheckerSettings.engine.none())
 		m_shouldRun = false;
+
+	auto const& ignoreCex = m_reader.stringSetting("SMTIgnoreCex", "no");
```

Comment:
@axic this option did not exist before, it is introduced into the test framework in this PR